services/app-api/src/postgres/questionResponse/insertQuestion.ts

```diff
@@ -1,7 +1,11 @@
 import { PrismaClient } from '@prisma/client'
-import { CMSUserType, StateUserType } from '../../domain-models'
+import { CMSUserType, QuestionResponseType } from '../../domain-models'
 import { convertPrismaErrorToStoreError, StoreError } from '../storeError'
-import { Question, CreateQuestionInput } from '../../domain-models'
+import {
+    Question,
+    CreateQuestionInput,
+    DivisionType,
+} from '../../domain-models'
```

Comment:
we're importing from domain models twice; we can combine these with the imports on line 2